enumerable.rb

```diff
@@ -0,0 +1,99 @@
+module Enumerable
+
+    def my_each
```

Comment:
* [ ] my_each should return enumerator when no block is given
```p [1,2,3,4,5].each{ |x| x}``` should ```return [1,2,3,4,5]```
[view ruby documentation for the functionality of ```.each```](https://ruby-doc.org/core-2.7.0/Enumerable.html)
Use this approach in other instances of your implementation.
ie Check functionality of the actual method from the ruby doc. You will have examples associated with each method in the doc, validate each custom method by making them pass the examples associated with the actual method.

Reply:
Awesome, thank you Addo, I will make those suggested changes.